grails-app/services/au/org/ala/biocollect/merit/AdminService.groovy

```diff
@@ -195,8 +195,6 @@ class AdminService {
 
     def syncCollectoryOrgs() {
         def url = "${grailsApplication.config.ecodata.service.url}admin/syncCollectoryOrgs"
-        webService.doPost(url, [
-                api_key: grailsApplication.config.api_key
-        ])
+        webService.doPost(url)
```

Comment:
doPost with single argument does not exist.

Reply:
Fixed in 14fc0d1db14c53cf1143e63d87ffe42f8fd13313